core/src/main/java/org/apache/iceberg/util/JsonUtil.java

```diff
@@ -162,7 +162,10 @@ public static Set<Integer> getIntegerSetOrNull(String property, JsonNode node) {
     if (!node.has(property) || node.get(property).isNull()) {
       return null;
     }
+    return getIntegerSet(property, node);
```

Comment:
Nit: we would normally include an empty newline between the control flow block and the following statement.